_data/faq.yml

```diff
@@ -177,7 +177,9 @@
   questions:
     - question: Kann ich als Knotenbetreiber_in dafür verantwortlich gemacht werden was andere über mein Freifunk tun? (Störerhaftung)
       answer: |
-        Nein! Der Datenverkehr wird verschlüsselt durch sogenannte Gateway Server getunnelt. Dadurch ist ist der Datenverkehr nicht zu Deinem Anschluss zurückverfolgbar und die sogenannte 'Störerhaftung' nicht durchsetzbar.
+        Nein! Die Internetverbindung, an die der Knoten angeschlossen ist, wird nicht für die Nutzer von Freifunk freigegeben.
```

Comment:
Dass die Leitung nicht mehr identifziert werden kann, ist ein Nebeneffekt, aber in diesem Moment nicht spezifisch die Absicht oder das "Verkaufsargument". Und an dieser Stelle sehe ich bei der alten Formulierung ein wenig Unseriösität. Hier wird von einer Umgehung der Störerhaftung gesprochen. Das haben wir nicht nötig und ein Verweis auf das Providerprivileg ist hier angebracht.

Vorschlag:
```
Nein! Die Internetverbindung, an die der Knoten angeschlossen ist, wird nicht direkt für die Nutzer von Freifunk freigegeben. Diese wird nur genutzt um einen Tunnel (VPN-Verbindung) zu unseren Servern herzustellen über die der Knoten auch mit allen Knoten außerhalb seiner Sendereichweite kommunizieren kann.
        Um den Nutzern_innen im Freifunk Zugriff auf Dienste des Internets zu geben, werden unsere Internetverbindungen an unseren Servern verwendet. Alle etwaigen Abmahnungen gehen an uns. Um uns vor diesen zu schützen verwenden wir das Providerprivileg, genau wie die Telekom, Vodafone oder andere große WLAN-Hotspot-Betreiber.
```